<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>464</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HomePage</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#contentPanel
{
	background: #fff;
}

QLabel#bottomLine1
{
	background: #4389b9
}

QLabel#bottomLine2
{
	background: #54abe7;
}

QWidget#buttonsPanel
{
	background: #fff;
}

QLabel#logoLabel
{
	border-image: url(:/logo.png) 0 0 0 0 stretch stretch;
}

QLabel#logoLabel2
{
	border-image: url(:/logo.png) 0 0 0 0 stretch stretch;
}



QLabel#buttonBackgroundLabel
{
	border-image: url(:/start_bg.png) 0 0 0 0 stretch stretch;
}

QWidget#testSelectionPanel
{
	border-style: solid;
	border-color: #ddd;
	border-width: 0px 0px 0px 1px;
	background: #fff;
}

QWidget#topBar
{
	background-color: #2a5674;
}

QLabel#selectorTitleLabel
{
	color: #fff;
	font-weight: bold;
	font-size: 16px;
}

QPushButton#selectorCloseButton,#selectorStartButton {
	background-color: #2a5674;
	border: none;
	min-width: 70px; max-width: 70px;
	color: #fff;
	font-weight: normal;
	font-size: 16px;
}

QPushButton#selectorCloseButton:hover,#selectorStartButton:hover
{
	background-color: #2d5c7c;
}

QPushButton#selectorCloseButton:pressed,#selectorStartButton:pressed
{
	background-color: #27516d;
}

QPushButton#selectorStartButton:disabled
{
	color: #54abe7;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="contentPanel" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="controlPanel" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>347</width>
               <height>110</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QWidget" name="buttonsWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>388</height>
              </size>
             </property>
             <widget class="PushButton" name="startButton">
              <property name="geometry">
               <rect>
                <x>89</x>
                <y>147</y>
                <width>166</width>
                <height>166</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#startButton
{
	color: #000;
	font-weight: normal;
	font-size: 28px;
	border-image: url(:/start.png) 0 0 0 0 stretch stretch;
}

QPushButton#startButton:hover
{
	border-image: url(:/start_hover.png) 0 0 0 0 stretch stretch;
}

QPushButton#startButton:pressed
{
	
	border-image: url(:/start_pressed.png) 0 0 0 0 stretch stretch;
}

</string>
              </property>
              <property name="text">
               <string>Start all</string>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
             <widget class="PushButton" name="selectButton">
              <property name="geometry">
               <rect>
                <x>223</x>
                <y>280</y>
                <width>92</width>
                <height>92</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>-1</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#selectButton
{
	color: #fff;
	font-weight: normal;
	font-size: 20px;
	border-image: url(:/select.png) 0 0 0 0 stretch stretch;
}

QPushButton#selectButton:hover
{
	border-image: url(:/select_hover.png) 0 0 0 0 stretch stretch;
}

QPushButton#selectButton:pressed
{
	border-image: url(:/select_pressed.png) 0 0 0 0 stretch stretch;
}</string>
              </property>
              <property name="text">
               <string>Test select</string>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
             <widget class="QLabel" name="logoLabel">
              <property name="geometry">
               <rect>
                <x>65</x>
                <y>2</y>
                <width>244</width>
                <height>82</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>244</width>
                <height>82</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>244</width>
                <height>82</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QLabel" name="buttonBackgroundLabel">
              <property name="geometry">
               <rect>
                <x>79</x>
                <y>137</y>
                <width>245</width>
                <height>245</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>245</width>
                <height>245</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>245</width>
                <height>245</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <zorder>buttonBackgroundLabel</zorder>
             <zorder>startButton</zorder>
             <zorder>selectButton</zorder>
             <zorder>logoLabel</zorder>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="apiSelectorContent">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>347</width>
               <height>100</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="apiSelectButton">
             <property name="styleSheet">
              <string notr="true">QPushButton {
	background-color: #54abe7;
	border: none;
	min-height: 40px; max-height: 40px;
	color: #ffffff;
}
</string>
             </property>
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="versionLabel">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Version</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="testSelectionPanel" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="topBar" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>44</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>44</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="selectorCloseButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Close</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="selectorTitleLabel">
              <property name="text">
               <string>Select Test</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="selectorStartButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Start</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="ItemScrollWidget" name="testSelectScrollWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="DescriptionWidget" name="descriptionPanel" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="bottomLine1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/res/homePage.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="bottomLine2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>2</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>2</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/res/homePage.png</pixmap>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DescriptionWidget</class>
   <extends>QWidget</extends>
   <header location="global">DescriptionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header location="global">PushButton.h</header>
  </customwidget>
  <customwidget>
   <class>ItemScrollWidget</class>
   <extends>QWidget</extends>
   <header>ItemScrollWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
